app.js

```diff
@@ -71,6 +71,17 @@ define(function(require) {
 					}
 				}
 			},
+			navigationMenus: [
+				'myOffice',
+				'numbers',
+				'users',
+				'groups',
+				'strategy',
+				'callLogs',
+				'devices',
+				'vmboxes',
+				'featureCodes'
+			],
```

Comment:
Move `navigationMenu` within `common`:
```suggestion
				},
				navigationMenus: [
					'myOffice',
					'numbers',
					'users',
					'groups',
					'strategy',
					'callLogs',
					'devices',
					'vmboxes',
					'featureCodes'
				]
			},
```